File: 07-04-network-image-viewmodel/java-07-04-network-image-viewmodel/app/src/main/java/com/example/pagingWithDatabinding/BindingAdapters.java
package com.example.pagingWithDatabinding;

import android.widget.ImageView;

import androidx.databinding.BindingAdapter;

import com.bumptech.glide.Glide;

public class BindingAdapters {
    @BindingAdapter("imgUrl")
    public static void setImgSrc(ImageView imageView, String imgUrl) {
        Glide.with(imageView.getContext())
                .load(imgUrl)
                .into(imageView);
    }
}

File: 07-04-network-image-viewmodel/kotlin-07-04-network-image-viewmodel/app/src/main/java/com/example/pagingWithDatabinding/BindingAdapters.kt
package com.example.pagingWithDatabinding

import android.widget.ImageView
import androidx.databinding.BindingAdapter
import com.bumptech.glide.Glide

object BindingAdapters {
    @BindingAdapter("imgUrl")
    @JvmStatic
    fun setImgSrc(imageView: ImageView, imgUrl: String?) {
        Glide.with(imageView.context)
                .load(imgUrl)
                .into(imageView)
    }
}